kops

```diff
@@ -176,15 +178,15 @@ client:
   client_cert:
   client_key:
   ssl_no_validate: False
-  http_auth: {}:{}
+  http_auth: {}{}{}
   timeout: 30
   master_only: False
 logging:
   loglevel: INFO
   logfile:
   logformat: default
   blacklist: ['elasticsearch', 'urllib3']
-""".format(logging_secrets['esUsername'], logging_secrets['esPassword'])
+""" .format(curator_secrets['esUsername'], ':' if curator_secrets["authentication"] else None, curator_secrets['esPassword'])
```

Comment:
Isn't it cleaner to use `''` instead of `None` here? I'm not even sure what the semantics is here for passing None instead of a string.